### ISSUE TITLE
[Compatibility Issue] SyntaxError: Invalid regular expression

### ISSUE THREAD

**Issue by ulivz** [yenshih/style-resources-loader#20]
```
SyntaxError: Invalid regular expression: /@(?:import|require)\s+(?:\([a-z,\s]+\)\s*)?['"]?(?<pathToResource>[^'"\s;]+)['"]?;?/: Invalid group
```

Which is introduced by https://github.com/yenshih/style-resources-loader/commit/c5fa23259a9f25b67c1be06bcf653e61c7ee0df3.

`(?<pathToResource>` (named capture groups) is supported from Node.js 10.3, see: https://node.green/

**Comment by leinov** [comment]
I have the same problem after the version upgrade
So I locked in version 1.2.1

**Comment by yenshih** [comment]
Definitely a bug. I'll fix it.

**Comment by yenshih** [comment]
Done. I've just released v1.3.1.
